### PR TITLE
Add demo: mobilenet taking image as input

### PR DESCRIPTION
**Pull request by gusye1234**

### CODE REVIEW

**Review by q82419 — CHANGES_REQUESTED** [review]
Please rebase to the lastest master.

File: examples/mobilenet/.gitignore
Comment (gusye1234) [reply examples/mobilenet/.gitignore:1]: But if user uses this demo, it will download fixtures under this dir, and mess git's tracking.

Comment (q82419) [reply examples/mobilenet/.gitignore:1]: It's user's responsibility to not to mess the source directory. After all, we'll not approve the PR with garbage files.

File: examples/mobilenet_image/download_mobilenet.sh
Comment (q82419) [inline examples/mobilenet_image/download_mobilenet.sh:11]: Use the original one: 
https://github.com/bytecodealliance/wasi-nn/tree/main/rust/images
Not to use the forked one.

File: examples/mobilenet_image/README.md
Comment (q82419) [inline examples/mobilenet_image/README.md:41]: Use the original one:
https://github.com/bytecodealliance/wasi-nn/tree/main/rust/images
Not to use the forked one.

**Review by q82419 — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (q82419) [inline .gitignore:6]: Need to add `openvino-mobilenet-raw/` ?

**Review by q82419 — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (gusye1234) [reply .gitignore:6]: OK

**Review by q82419 — APPROVED** [review]

**Pull request merged**
